include/mbedtls/cipher.h

```diff
@@ -96,8 +96,8 @@ typedef enum {
  *            ciphers instead.
  */
 typedef enum {
-    MBEDTLS_CIPHER_NONE = 0,             /**< None. */
-    MBEDTLS_CIPHER_NULL,                 /**< NULL. */
+    MBEDTLS_CIPHER_NONE = 0,             /**< Placeholder to mark the end of cipher-pair lists. */
+    MBEDTLS_CIPHER_NULL,                 /**< The cipher-pair, treated as a stream cipher. */
```

Comment:
Suggestion: Change to `The identity stream cipher.` There's no mode information to add here.